src/Microsoft.SqlTools.ManagedBatchParser/Microsoft.SqlTools.ManagedBatchParser.csproj

```diff
@@ -1,6 +1,5 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Deleted the property only for ManagedBatchParser! remaining projects still targeting to what they have! let me know if you want to apply it for all other projects.